### ISSUE TITLE
LR schedule

### ISSUE THREAD

**Issue by XinyangHan** [cvlab-columbia/zero123#50]
Hi! @ruoshiliu 

May I ask a question about lr schedular? Currently it seems you are using constant lr=1 after warm up. Is it the optimal schedule you've found? I am asking this question because I am wondering what would be the appropriate lr if we want to fine tune the model without deviating.

Thanks so much for your help!

**Comment by ruoshiliu** [comment]
Hi @XinyangHan , it's a good question. I don't think constant lr=1e-4 is optimal but we didn't really have the time and resources to tune the learning rate here -- we are talking about weeks of time just to run one experiment. I'm sure the parameter can be furthered tuned to improve the convergence rate so I would appreciate it if you find something better and let me know :)

**Comment by XinyangHan** [comment]
Got it! Also, in the appendix C1, the paper mentioned that the variance between batches would cause the model converging slower? Does that means that whether training or finetuning zero123, we might want a really large batchsize e.g.  1536?

**Comment by ruoshiliu** [comment]
Yes, large batch size is quite important. We used 1536 but it might not be necessary. 512 and a smaller learning rate might also work?

**Comment by XinyangHan** [comment]
Got it!! Thanks for helping!!

**Comment by EricLee0224** [comment]
Hi @XinyangHan , I am also working on the finetune the zero123 model, would it be possible for you to provide contact info so that I can discuss this with you? Many thx!

**Comment by yuanzhi-zhu** [comment]
Hi @EricLee0224 , FYI, I tried to finetune the zero123 model with lora and could not get satisfying results

**Comment by fradif96** [comment]
Hello @EricLee0224 @yuanzhi-zhu 
did you finally somehow manage fine-tuning of zero123 model ?
Thanks in advance

**Comment by yuanzhi-zhu** [comment]
> Hello @EricLee0224 @yuanzhi-zhu did you finally somehow manage fine-tuning of zero123 model ? Thanks in advance

Hi @fradif96 , No, I did not. I suspect that it is not possible to make SD 3D-aware by finetuning.
